### PR TITLE
Fake Players shoot at pretty much anything they see.

### PR DESCRIPTION
**Pull request by peterimprobable**
Main issue is the lack of pitch when they aim.
This is due to the anim blueprint using the controller remote pitch value, which would first of all read from the GDK Player Controller, not the non-rep'd SimPlayer controller, and secondly the Controller SetFocus method only controls yaw, not pitch.
This is something we will need to fix for AI anyway, so suggest including it as is for now.

### CODE REVIEW

File: Game/Source/GDKShooter/Private/Characters/Components/NPCFunctionLibrary.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		float Strength;
+		FVector ViewLocation;
+		FRotator ViewRotation;
+		SelfAsPawn->GetActorEyesViewPoint(ViewLocation, ViewRotation);
+
+		if (Sight->CanBeSeenFrom(ViewLocation, Location, LoSChecks, Strength, SelfAsPawn))
+		{
+			return Strength > 0.f;
+		}
+		else {
```
Comment (m-samiec) [inline Game/Source/GDKShooter/Private/Characters/Components/NPCFunctionLibrary.cpp]:
```suggestion
		else 
		{
```

**Comment by girayozil** [comment]
Could you paste screenshots for controller and character BP classes?

**Comment by peterimprobable** [comment]
image

**Comment by peterimprobable** [comment]
image
image

**Pull request closed without merging**